infrastructure/kubernetes/app-backend.yml

```diff
@@ -28,21 +28,27 @@ spec:
         app: app-backend
     spec:
       containers:
-      - image: "teamdigitale/italia-backend:v0.0.21"
+      - image: "teamdigitale/italia-backend:v0.0.23"
         name: app-backend
         ports:
         - name: http-port
           containerPort: 80
         env:
         - name: "ALLOW_NOTIFY_IP_SOURCE_RANGE"
           value: "0.0.0.0/0"
+        - name: "ALLOW_PAGOPA_IP_SOURCE_RANGE"
```

Comment:
you should ask SIA to give us their source IP range (perhaps create a story for this update)